compiler/modules/print.llvm/common.bal

```diff
@@ -85,6 +85,8 @@ type BinaryOp IntOp|FloatArithmeticOp;
 // Corresponds to LLVMIntPredicate
 public type IntPredicate "eq"|"ne"|"ugt"|"uge"|"ult"|"ule"|"sgt"|"sge"|"slt"|"sle";
 
+// Corresponds to LLVMRealPredicate
+public type RealPredicate "false"|"oeq"|"ogt"|"oge"|"olt"|"ole"|"one"|"ord"|"ueq"|"ugt"|"uge"|"ult"|"ule"|"une"|"uno"|"true";
```

Comment:
I am renaming `RealType` to `BinaryFp` as suggested in [#306 (comment)](https://github.com/ballerina-platform/nballerina/pull/306#discussion_r688860125).
Let's change this to `BinaryFpPredicate` to match.

Reply:
Our LLVM module is using terminology from the LLVM lang ref (in preference to LLVM API) and the lang ref calls these floating point types https://llvm.org/docs/LangRef.html#floating-point-types, so I think maybe FpType or FloatingPointType is better.

Reply:
Binary is going to get confused with Binary operators.

Reply:
Looking at the code I think `FloatType` will work best.

Reply:
Renamed to `FloatPredicate` to match `FloatType`